src/main.rs

```diff
@@ -22,7 +25,7 @@ mod callbacks;
 mod lua;
 
 /// Callback to route wlc logs into env_logger
-extern "C" fn log_handler(level: LogType, message_ptr: *const libc::c_char) {
+extern "C" fn log_handler(level: LogType, message_ptr: *const i8) {
```

Comment:
[It isn't for arm](https://doc.rust-lang.org/libc/arm-unknown-linux-gnueabihf/libc/type.c_char.html). libc should deal with platform-specific issues with dealing with C. If we keep it in its current state and try to compile on ARM (say for a raspberry pi) then we will get a compile error.